packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -126,6 +132,7 @@ const SearchInputBase: React.FunctionComponent<SearchInputProps> = ({
   const [searchValue, setSearchValue] = React.useState(value);
   const searchInputRef = React.useRef(null);
   const searchInputInputRef = innerRef || React.useRef(null);
+  const [isSearchInputExpanded, setIsSearchInputExpanded] = React.useState(true);
```

Comment:
I think it could be removed. The state could be used to enable uncontrolled expansion, but as is it doesn't appear to work after playing with the example (`onToggleExpand` is required to turn the input into a expandable variant, and removing `isExpanded` prevents expansion entirely).